### ISSUE TITLE
Typological information display when clicking on the map

### ISSUE THREAD

**Issue by dhakelila** [Vizzuality/mangrove-atlas#426]
As discussed long ago, when the user clicks on the restoration layer, they will get additional information about those areas (typological units) in a tooltip. 

This post aims to clarify the details and align both teams on what will be implemented. Please, @klongleywood , @lhilarides let us know if you agree with the description below. 

(1) The behaviour
- User clicks on the map
- A tooltip shows information related to that area. 
- The area gets highlighted. 
- To close the panel, we'll include a "X" icon

(2) The info to show (and how)
As per the mock-up shared by Lammert, we'll show:
- Mangrove type (figure), 
- Max Mangrove area 1996 - 2016 (figure),
- Area loss (figure),
- Restorable area (figure),
- Percentage restorable (figure),
- Area degraded (figure),
- Restoration values (the chart with horizontal bars),
- Restoration potential (donut chart),
- Ecosystem Services Value for restored Mangroves (we'll display only the data available)
- - Soil organic carbon,
- - Aboveground Carbon (figure), 
- - People protected (figure), 
- - Commercial Fish... (figure), 
- - Commercial Invert (figure)

![Screenshot 2022-07-11 at 22 37 37](https://user-images.githubusercontent.com/6608294/178353740-61a716c2-e8de-4ed4-ba9e-90ff802ab4a7.png)

Link to the file: https://drive.google.com/file/d/1Dy1RyjV_rZSpD_lwfLFgTlq4cgzhM8YB/view?usp=sharing

**Comment by klongleywood** [comment]
Hi both -- I agree with Lammert's comments and wanted to add that we won't have the degradation data ready, so that metric should be removed.

**Comment by dhakelila** [comment]
@klongleywood this was done.

**Comment by klongleywood** [comment]
@dhakelila in the Mangrove Degradation and Loss infographic, the third statistic should be restorable area (that's what it should have been instead of degraded area)

**Comment by dhakelila** [comment]
@klongleywood , @AngelArcones do we have that data (restorable area)?
I never understood we should add that but only remove degraded area.

**Comment by klongleywood** [comment]
@dhakelila @AngelArcones Yes, we have that data -- it's actually the same field used under the Restorable Mangrove Area heading (Total Restorable Area) but showing that as a proportion of total loss is part of the story that not all lost areas are restorable.

**Comment by dhakelila** [comment]
![Screenshot 2022-09-12 at 22 25 42](https://user-images.githubusercontent.com/6608294/189752092-beb1583e-f96f-4d49-bf25-4995d1091172.png)

Ready for revision

**Comment by klongleywood** [comment]
@dhakelila here it is important that the restorable area is a subset of the total area of loss. We need to convey that somehow. My suggestion is that the Lost area should be re-labeled as "Non-Restorable Lost Mangrove Area" and that the value should be the area of loss - the restorable area value. Open to other ideas though

**Comment by dhakelila** [comment]
![Screenshot 2022-09-16 at 12 28 19](https://user-images.githubusercontent.com/6608294/190619166-9f5b8a57-d257-480b-8653-16522fd63a64.png)

Fixed. Please, check the numbers.
